views/admin/requests/index.js

```diff
@@ -239,8 +239,8 @@ var getResults = function(callback) {
     };
 
     var createCSV = function() {
-        var requestFieldNames = ['id','name','address','city','state','zip','phone','email','date created','region'];
-        var requestFields = ['serial','name','address','city','state','zip','phone','email','createdAt','assigned_rc_region'];
+        var requestFieldNames = ['id','name','address','city','state','zip','phone','email','date created','region', 'origin'];
```

Comment:
The mismatch between "origin" in the field name and "source" in the request bugs me.  I guess that mismatch existed already, and shouldn't be fixed as part of this change, but would it be hard to fix afterwards in a separate change?